expression/column.go

```diff
@@ -140,6 +140,12 @@ func (col *CorrelatedColumn) ResolveIndices(_ *Schema) Expression {
 func (col *CorrelatedColumn) resolveIndices(_ *Schema) {
 }
 
+// ColumnIdentifier represents a identifier of column.
+type ColumnIdentifier struct {
```

Comment:
Can you wait for this merged?
https://github.com/pingcap/tidb/pull/7157

Reply:
@winoros should #7157 cherry-pick to 2.0.6~?

Reply:
The answer might be `no` for now.